dvc/remote/gdrive.py

```diff
@@ -21,21 +21,27 @@
 
 
 class GDrivePathNotFound(DvcException):
-    def __init__(self, path_info):
-        super().__init__("Google Drive path '{}' not found.".format(path_info))
+    def __init__(self, path_info, hint):
+        hint = "" if hint is None else " {}".format(hint)
+        super().__init__(
+            "GDrive path '{}' not found.{}".format(path_info, hint)
+        )
 
 
-class GDriveAccessTokenRefreshError(DvcException):
-    def __init__(self):
-        super().__init__("Google Drive access token refreshment is failed.")
+class GDriveAuthError(DvcException):
+    def __init__(self, cred_location):
 
+        if cred_location:
+            message = (
+                "GDrive remote auth failed with credentials in '{}'.\n"
+                "Backup first, remove of fix it, and run the command.\n"
+                "It should do auth again and refresh the credentials.\n\n"
```

Comment:
```suggestion
                "Remove of fix it, and run the command again.\n\n"
```

Reply:
again ... again (repetition with the next sentence)

`Backup ..` - I think it's better to include it. What is your concern here?

Reply:
Just trying to simplify. Shouldn't they decide whether to back it up? Why back it up if it's not working?
As for "again", I don't see much problem in the repetition but actually I would just remove the last sentence altogether, it may be obvious that you'll go through the same process when you run the command again.

Up to you, I don't have string opinions here.

Reply:
Resolving, I think it should be fine. And it's def better to ask to backup - I don't want to take any responsibility for someone deleting some credentials by mistake.